isis/tests/AlternativeTestCsmModel.cpp

```diff
@@ -4,8 +4,27 @@
 #include <nlohmann/json.hpp>
 using json = nlohmann::json;
 
+const std::string AlternativeTestCsmModel::SENSOR_MODEL_NAME = "TestCsmModelName";
+const std::vector<std::string> AlternativeTestCsmModel::PARAM_NAMES = {
```

Comment:
Not sure if it matters here, but I had this one have TestParams1->3, not just 1 and 2.